docs/new/posts/PostgreSQL/postgresql-q4-2025-release.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2025-12-09
+description: >
+  Percona Distribution for PostgreSQL 17.7, 16.11, 15.15, 14.20, 13.23 (EOL) have been released.
+authors: [Andriciuc]
+categories:
+  - PostgreSQL
+tags:
+  - Percona Distribution for PostgreSQL
+---
+
+# Percona Distribution for PostgreSQL 17.7, 16.11, 15.15, 14.20, 13.23 (EOL) have been released
```

Comment:
Is this one distribution (one thing), then the verb should be "has"
If these are multiple distributions then the verb is "have".
I guess I'm confused with the singular Distribution for multiple releases.